src/components/TodoModal/TodoModal.tsx

```diff
@@ -15,25 +40,36 @@
               className="modal-card-title has-text-weight-medium"
               data-cy="modal-header"
             >
-              Todo #2
+              Todo #{todo.id}
             </div>
 
-            {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-            <button type="button" className="delete" data-cy="modal-close" />
+            <button
+              type="button"
+              className="delete"
+              data-cy="modal-close"
+              onClick={() => handleSetSelectedTodoId(0)}
```

Comment:
The method name `handleSetSelectedTodoId` does not follow the naming conventions for event handlers in React. According to the conventions, the method should be named `onSetSelectedTodoId` or `onTodoIdChange`.